readme.md

```diff
@@ -3,8 +3,7 @@
 ![Els_kom Icon](./els_kom.png)
 
 ![.NET Core](https://github.com/Elskom/Els_kom_new/workflows/.NET%20Core/badge.svg?branch=main&event=push)
-![.NET Core](https://github.com/Elskom/Els_kom_new/workflows/.NET%20Core/badge.svg?event=pull_request)

```

Comment:
I think a better option would be to split the workflows for prs and normal default branch build like what the Sdk repository does.

Reply:
> I think a better option would be to split the workflows for prs and normal default branch build like what the Sdk repository does.

I don't see any reason to make a badge for any PR. While on PR, a contributor will anyway see the current status.

Reply:
True, it would be nice to actually instead have a badge that says ``x pr's passing, y pr's failing`` where x and y are numbers of passing or failing prs.

Reply:
That's really useless imho 😃 . I also doubt there's a badge for this case.

Reply:
Feel free to push to my fork, if you want.

Reply:
Ok, while messing around in the Sdk I found this:
```
[![Github pulls-passing](https://img.shields.io/github/issues-search/Elskom/Sdk?label=passing%20pull%20requests&query=is%3Aopen+status%3Asuccess+is%3Apr&color=limegreen)](https://github.com/Elskom/Sdk/pulls?q=is%3Aopen+status%3Asuccess+is%3Apr)
[![Github pulls-failing](https://img.shields.io/github/issues-search/Elskom/Sdk?label=failing%20pull%20requests&query=is%3Aopen+status%3Afailure+is%3Apr&color=red)](https://github.com/Elskom/Sdk/pulls?q=is%3Aopen+status%3Afailure+is%3Apr)
```